slamd/discovery/processing/experiment/models/tuned_random_forest.py

```diff
@@ -0,0 +1,42 @@
+from slamd.discovery.processing.experiment.models.slamd_random_forest import SlamdRandomForest
+
+from mlxtend.feature_selection import SequentialFeatureSelector
+from sklearn.model_selection import GridSearchCV
+from sklearn.pipeline import Pipeline
+
+
+class TunedRandomForest:
+
+    @classmethod
+    def find_best_model(cls, training_rows, training_labels):
+        # Perform a forward sequential feature selection with 3 features without cross-validation
```

Comment:
Here and for Tuned Gauss. We actually never perform a seq. feat. selection with 3 features. It is overwritten by the values in our grid which are 5 and 10, respectively. I guess we could init we the default value here. It is just an artefact that we have a 3 here. Please check if you can use the default and it still works.